src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -6285,8 +6498,53 @@ public void testListBusinessStakeholders_withUserTokenAndSomeOptions() throws Ex
+        Mockito.verify(mockApiClient).get(Mockito.eq("https://api.sandbox.hyperwallet.com/rest/v4/users/" + token
+                + "/business-stakeholders?createdBefore=2016-06-29T17:58:26Z&limit=10"), Mockito.any(TypeReference.class));
+
+    }
+
+    @Test
+    public void testDocumentUpload_noUserToken() {
+        Hyperwallet client = new Hyperwallet("test-username", "test-password");
+        try {
+            client.documentUpload(null, new FormDataMultiPart());
+            fail("Expect HyperwalletException");
+        } catch (HyperwalletException e) {
+            assertThat(e.getErrorCode(), is(nullValue()));
+            assertThat(e.getResponse(), is(nullValue()));
+            assertThat(e.getErrorMessage(), is(equalTo("User token is not present")));
+            assertThat(e.getMessage(), is(equalTo("User token is not present")));
+            assertThat(e.getHyperwalletErrors(), is(nullValue()));
+            assertThat(e.getRelatedResources(), is(nullValue()));
+        }
+    }
+
+    @Test
+    public void testDocumentUpload_successful() throws Exception {
+        try {
+            Hyperwallet client = new Hyperwallet("test-username", "test-password");
 
+            HyperwalletUser hyperwalletUser = new HyperwalletUser();
+            DocumentVerificationDocumentsRepresentation documentVerificationDocumentsRepresentation =
+                    new DocumentVerificationDocumentsRepresentation();
+            documentVerificationDocumentsRepresentation.category(EDocumentCategory.AUTHORIZATION)
+                    .type(EIdentityVerificationType.LETTER_OF_AUTHORIZATION)
+                    .country(ECountryCode.CA).status(EKycDocumentVerificationStatus.NEW);
+            List<DocumentVerificationDocumentsRepresentation> documentVerificationDocumentsRepresentationList = new ArrayList<>();
+            documentVerificationDocumentsRepresentationList.add(documentVerificationDocumentsRepresentation);
+            hyperwalletUser.setDocuments(documentVerificationDocumentsRepresentationList);
+
+            HyperwalletApiClient mockApiClient = createAndInjectHyperwalletApiClientMock(client);
+            Mockito.when(mockApiClient.put(Mockito.anyString(), Mockito.any(FormDataMultiPart.class), Mockito.any(Class.class)))
+                    .thenReturn(hyperwalletUser);
+
+            HyperwalletUser hyperwalletUserresponse = client.documentUpload("test-token", new FormDataMultiPart());
+            hyperwalletUserresponse.getDocuments().get(0).getCategory().equals(EDocumentCategory.AUTHORIZATION);
+            hyperwalletUserresponse.getDocuments().get(0).getType().equals(EIdentityVerificationType.LETTER_OF_AUTHORIZATION);
+            hyperwalletUserresponse.getDocuments().get(0).getCountry().equals(ECountryCode.CA);
+            hyperwalletUserresponse.getDocuments().get(0).getStatus().equals(EKycDocumentVerificationStatus.NEW);
+        } catch (HyperwalletException e) {
+        }
```

Comment:
Catch block is empty

Reply:
Catch block removed and added assert statements